astropy/io/votable/tree.py

```diff
@@ -512,6 +515,11 @@ def parse(self, iterator, config):
     def to_xml(self, w, **kwargs):
         w.element(self._element_name, self._content,
                   attrib=w.object_attrs(self, self._attr_list))
+        
+    def to_meta(self):
+        d = dict([(key, getattr(self, key)) for key in self._attr_list])
```

Comment:
Is it "cleaner" to call parent's `to_meta` here or is that extra overhead? Not sure.